judge/views/mailgun.py

```diff
@@ -26,7 +28,7 @@ def post(self, request, *args, **kwargs):
 
             logger.debug('Received request: %s', params)
 
-            if signature != hmac.new(key=settings.MAILGUN_ACCESS_KEY, msg='%s%s' % (timestamp, token),
+            if signature != hmac.new(utf8bytes(key=settings.MAILGUN_ACCESS_KEY), msg=b'%s%s' % (timestamp, token),
```

Comment:
Uhhh... No. You can't just `b'%s%s'` when the arguments are strings. You have to format it normally and then `utf8bytes` it.